docs/management/admin/host-isolation-exceptions.asciidoc

```diff
@@ -14,20 +16,28 @@ IMPORTANT: Each host isolation exception IP address should be a highly trusted a
 .. `Name your host isolation exceptions`: Enter a name to identify the host isolation exception.
 .. `Description`: Enter a description to provide more information on the host isolation exception (optional).
 .. `Enter IP Address`: Enter the IP address for which you want to allow communication with an isolated host. This must be an IPv4 address, with optional CIDR notation (for example, `0.0.0.0` or `1.0.0.0/24`, respectively).
-. Click **Add Host isolation exception**.
+. Select an option in the *Assignment* section to assign the host isolation exception to a specific integration policy:
++
+* `Global`: Assign the host isolation exception to all integration policies for {endpoint-sec}.
+* `Per Policy` (Platinum or Enterprise subscription only): Assign the host isolation exception to one or more specific {endpoint-sec} integration policies. Select each policy in which you want the host isolation exception to be applied.
++
+NOTE: You can also select the `Per Policy` option without assigning a policy to the host isolation exception at this time. For example, you could do this if you want to create and review your host isolation exception configurations before putting them into action with a policy.
+. Click **Add Host isolation exception**. The new exception is added to the *Host isolation exceptions* list.
 
 [discrete]
 [[manage-host-isolation-exceptions]]
 == View and manage host isolation exceptions
 
-The **Host isolation exceptions** list displays all the host isolation exceptions that have been configured for {elastic-sec}. To refine the Host isolation exceptions list, enter a query in the search bar. You can search by name, description, or IP address.
+The **Host isolation exceptions** list displays all the host isolation exceptions that have been configured for {elastic-sec}. To refine the **Host isolation exceptions** list, enter a query in the search bar. You can search by name, description, or IP address.
 
 [role="screenshot"]
 image::images/host-isolation-exceptions-ui.png[List of host isolation exceptions]
 
 [discrete]
 [[edit-host-isolation-exception]]
 === Edit a host isolation exception
+You can change the configuration of each host isolation exception. If you have a Platinum or Enterprise subscription, you can also change the policy assigned to a host isolation exception.
```

Comment:
```suggestion
You can individually configure each host isolation exception. If you have a Platinum or Enterprise subscription, you can also change the policy or policies assigned to each host isolation exception.
```
Are policies assigned to exceptions or exceptions assigned to policies? I'm inclined to think it's the latter, but not sure how much it matters.